packages/client-cli/test/frontend-openapi.test.mjs

```diff
@@ -636,11 +637,8 @@ test('integration test for custom headers', async (t) => {
   const testFile = `
 'use strict'
 
-import build, { setBaseUrl, setDefaultHeaders, getReturnHeaders } from './foobar.mjs'
-const client = build('${app.url}')
-// const client = build('https://echo.free.beeceptor.com')
-
-setDefaultHeaders({
+import build from './foobar.mjs'
+const client = build('${app.url}', {
   authorization: 'Bearer foobar'
```

Comment:
maybe let's wrap this behind an `headers` prefix. I suspect we'll add more configs here.